pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -93,10 +93,17 @@ class CatDogImageDataModule(LightningDataModule):
 
     def __init__(
         self,
-        dl_path: Union[str, Path],
+        dl_path: Union[str, Path] = "data",
         num_workers: int = 0,
         batch_size: int = 8,
     ):
+        """CatDogImageDataModule
+
+        Args:
+            dl_path: Root directory where to download the data
+            num_workers: number of CPU workers
+            batch_size: number of sample in a batch
```

Comment:
```suggestion
            dl_path: root directory where to download the data
            num_workers: number of CPU workers
            batch_size: number of sample in a batch
```
inconsistent capitalization